src/pipeline.rs

```diff
@@ -139,24 +139,18 @@ impl Plugin for AtmospherePipelinePlugin {
             .init_resource::<CachedAtmosphereModelMetadata>()
             .init_resource::<AtmosphereImageBindGroupLayout>()
             .init_resource::<Events<AtmosphereUpdateEvent>>()
-            .add_system_to_stage(RenderStage::Extract, extract_atmosphere_resources)
-            .add_system_to_stage(
-                RenderStage::Prepare,
-                Events::<AtmosphereUpdateEvent>::update_system,
-            )
-            .add_system_to_stage(
-                RenderStage::Prepare,
+            .add_system(extract_atmosphere_resources.in_schedule(ExtractSchedule))
+            .add_system(Events::<AtmosphereUpdateEvent>::update_system.in_set(RenderSet::Prepare))
+            .add_system(
                 prepare_atmosphere_assets
-                    .label(PrepareAssetLabel::PostAssetPrepare)
-                    .after(PrepareAssetLabel::AssetPrepare),
+                    .in_set(PrepareAssetSet::PostAssetPrepare)
+                    .after(PrepareAssetSet::AssetPrepare),
```

Comment:
I don't think this `.after()` is actually necessary

Reply:
Yes, probably a sanity check. I'll remove it.